projects/npm-tools/packages/npm-scripts/src/utils/bundlerNamespace.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * SPDX-FileCopyrightText: © 2019 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+function addNamespace(packageName, namespacingPackageName) {
+	const namespace = makeNamespace(namespacingPackageName);
+
+	if (packageName.indexOf('@') === 0) {
+		return `@${namespace}$${packageName.substr(1)}`;
```

Comment:
Yeah, the guys at webpack also like that change. 

I really prefer `substr` or `substring`, since it is more evident than `slice` (at least to me) when you are dealing with strings. I mean, `slice`can be used for arrays too.